src/vg/civcraft/mc/civchat2/listeners/MercuryMessageListener.java

```diff
@@ -0,0 +1,48 @@
+package vg.civcraft.mc.civchat2.listeners;
+
+import java.util.UUID;
+
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.scheduler.BukkitRunnable;
+
+import vg.civcraft.mc.civchat2.CivChat2;
+import vg.civcraft.mc.mercury.events.AsyncPluginBroadcastMessageEvent;
+import vg.civcraft.mc.namelayer.NameAPI;
+import vg.civcraft.mc.namelayer.NameLayerPlugin;
+
+public class MercuryMessageListener extends BukkitRunnable implements Listener {
```

Comment:
This class does not need to be a runnable.  Just make it implement Listener and register it in the main class.  The issue in namelayer was that namelayer is programmed to try and be the first program running always.  Civchat should handle its dependencies fine since Mercury for sure will load before it.